views/index.erb

```diff
@@ -18,5 +18,29 @@
 			<input type="password" name="password" placeholder="Password" required>
 			<input type="submit">
 		</form>
-	</div>
+</div>
+</div>
+
+<div id="student_modal_background">
+</div>
+
+<div id="student_login_modal">
+  <div id="student_close"><a href="javascript:;">X</a></div>
+  <div id="title"><strong>Login</strong></div>
+  <div id="student_login">
+    <form action="/student_login" method="post">
+      <input type="text" name="username" placeholder="Username" required>
+      <input type="password" name="password" placeholder="Password" required>
+      <input type="submit">
+    </form>
+</div>
+<div>
+ <div id="title"><strong>Sign Up!</strong></div>
+    <form action="/student_signup" method="post">
+      <input type="text" name="username" placeholder="Username" required>
+      <input type= "text" name="github" placeholder="Github Account" required>
+      <input type="password" name="password" placeholder="Password" id = "signup" maxlength= "10" required>
```

Comment:
I think 10 is probably good for now? I guess these days a lot of passwords are longer, need to be. I think 10 is the character limit I set on the admin password too in the database table.